bot_modules/irc_commands.js

```diff
@@ -482,6 +482,67 @@ module.exports = {
 			})
 		})
 	},
+  
+	/**
+	 *  roll
+	 *
+	 */
+	roll: (info, words) => {
+		let dice_notation = words.slice(1).join(" ").toLowerCase();
+		if (dice_notation.includes("d")) {
+	   		if (dice_notation == "") dice_notation = "1d10";
```

Comment:
dice_notation will never be an empty string if it includes "d"

Reply:
it doesn't have to include "d". dice_notation is passed by user

Reply:
if dicenotation includes d then if dicenotation is empty

Reply:
no, it's if dice_notation includes d, then if it doesn't, in case user wants to do a basic roll, like `!roll 100`

Reply:
perhaps the naming is difficult to understand